pages/dkp/kommander/1.3/workspaces/workspace-platform-services/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: layout.pug
+beta: true
+navigationTitle: Workspace Platform Services
```

Comment:
fyi @cmastr @danielone renamed this page away from "Federated Platform Services" -> "Workspace Platform Services"